apps/condo/bin/delete-contacts-by-organization-id.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path')
+const { find } = require('@condo/keystone/schema')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { isEmpty } = require('lodash/isEmpty')
```

Comment:
I think you don't need object destruction if you using direct path to module
```suggestion
const isEmpty = require('lodash/isEmpty')
```
You can see how it was exported inside lodash module
<img width="189" alt="image" src="https://user-images.githubusercontent.com/25844927/196421595-04fa9931-42b0-465e-b384-96e80e870254.png">